src/typeMapper.js

```diff
@@ -68,12 +69,15 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
   if (sequelizeType instanceof FLOAT ||
       sequelizeType instanceof DOUBLE) return GraphQLFloat;
 
+  if (sequelizeType instanceof DATE ||
+      sequelizeType instanceof DATEONLY) {
```

Comment:
is DATEONLY parsed as a date objecti n Sequelize? Isn't it just a date string?

Reply:
I think it depends on the version and the latest is String. See https://github.com/sequelize/sequelize/issues/4858 I'll switch it back String to be on the safe side.